pages.config.ts

```
@@ -0,0 +1,43 @@
+  tabBar: {
+    color: '#999999',
+    selectedColor: '#018d71',
+    backgroundColor: '#F8F8F8',
+    borderStyle: 'black',
+    height: '50px',
+    fontSize: '10px',
+    iconWidth: '24px',
+    spacing: '3px',
+    list: [
+      {
+        iconPath: 'static/tab-bar/home.png',
```

ESLint

[error] Replace `'static/tab-bar/home.png'` with `"static/tab-bar/home.png"`

(prettier/prettier)